embedded/servo_bq/servo_balsaq.cc

```diff
@@ -0,0 +1,50 @@
+//%bin(servo_balsaq_main)
+#include "embedded/servo_bq/servo_balsaq.hh"
+#include "infrastructure/balsa_queue/bq_main_macro.hh"
+
+#include <chrono>
+#include <cstddef>
+#include <iostream>
+#include <sstream>
+#include <cstdlib>
+
+
+//%deps(balsa_queue)
+//%deps(message)
+
+namespace jet {
+
+ServoBq::ServoBq() {
+  set_comms_factory(std::make_unique<MqttCommsFactory>());
+}
+constexpr int PWM_FREQUENCY = 330;
+
+void ServoBq::init(int argc, char *argv[]) {
+  servo_index = atoi(argv[1]); // get the servo number in 0...3 from the command line
+
+  int i2cHandle = i2c_open("/dev/i2c-1");
+  if (i2cHandle == -1) {
+    std::cout << "Failed to open i2c" << std::endl;
+    // return -1;
+  }
+  std::shared_ptr<PwmDriver> pwm_driver = std::make_shared<PwmDriver>(i2cHandle);
```

Comment:
Make this a unique pointer, then move it into the servo driver's constructor, since you're not sharing the PwmDriver with anyone.